app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PagesController < ApplicationController
```

Comment:
Can we call this `StaticPagesController` instead?

Reply:
👍